pyproject.toml

```diff
@@ -42,6 +42,12 @@ conversion = [
     "requests>=2.26.0",
     "tqdm>=4.62.3",
 ]
+doc = [
+    "black>=24.1.1",                # required by mkdocs to format the signatures
```

Comment:
Any reason to have so many spaces (for the inline comment)?

Reply:
It got auto-formatted by [Even Better TOML](https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml) like this. I'll put the comment above the line rather than next to it.